calico_node/startup/startup.go

```diff
@@ -115,10 +87,16 @@ func main() {
 		terminate()
 	}
 
-	// Set other Felix config that is not yet in the node resource.
-	if err := ensureDefaultConfig(client, node); err != nil {
-		fatal("Unable to set global default configuration: %s", err)
-		terminate()
+	// Configure IP Pool configuration.
```

Comment:
Actually do we need a comment here, the name itself is pretty descriptive of what it does and the doctring sheds a bit more light for those curious could we just remove it?  Most of the comments in this section are like that.